### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by Justfun1517**
- [DEMO LINK](https://Justfun1517.github.io/layout_html-form/)
- [TEST REPORT LINK](https://Justfun1517.github.io/layout_html-form/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -13,5 +13,182 @@
...
+              name="recommend"
+            > Yes
+          </label>
+          <label>
+            <input 
+              type="radio" 
+              name="recommend"
+            > No
+          </label>
+        </label>
```
Comment (Bohdan-Maksymiuk) [inline src/index.html:181]:
```suggestion
          <label>
            <input 
              type="radio" 
              name="recommend"
            >
            No
          </label>
        </label>
```

File: src/index.html [hunk 2]
```diff
@@ -13,5 +13,182 @@
...
+
+      <button 
+        type="submit" 
+        class="submit"
+        name="button"
+        id="button"
+      >Submit
+      </button>
+  
+    </form>
```
Comment (Bohdan-Maksymiuk) [inline src/index.html:192]:
```suggestion
        id="button"
      >
        Submit
      </button>
    </form>
```

File: src/style.css [hunk 3]
```diff
@@ -1 +1,10 @@
 /* styles go here */
+.form-field{
+  display: block;
+}
+fieldset{
+  margin:20px 0px 20px 0px;
+}
+input{
+  margin-top:10px;
+}
```
Comment (Bohdan-Maksymiuk) [inline src/style.css:10]: use class selectors

File: src/style.css [hunk 4]
```diff
@@ -1 +1,10 @@
 /* styles go here */
+.form-field{
```
Comment (Bohdan-Maksymiuk) [inline src/style.css:2]:
```suggestion
.form-field {
```

**Review by Bohdan-Maksymiuk — CHANGES_REQUESTED** [review]
https://user-images.githubusercontent.com/94289038/199244298-51904294-9dc3-404c-b06c-677381837a86.mov

* you should add link as I show you every time

* your demo show me readme file and test demo - 404
check gh repo settings and deploy again


https://user-images.githubusercontent.com/94289038/199245846-bd39f1be-5a55-4737-838f-7ecb1a0e456b.mov



feel free to ask mentors in `fe_chat` if smth is unclear for you

**Comment by Justfun1517** [comment]
Thanks for your help! 
Found the problem and solved it. Now links work.

**Pull request closed without merging**